extensions/Cake.Vagrant.yml

```diff
@@ -8,4 +8,5 @@ Author: Alistair Chapman
 Description: Cake addin for creating and managing Vagrant virtual machines
 Categories:
 - vagrant
-TargetCakeVersion: 0.28.1
+TargetCakeVersion: 0.13.0
```

Comment:
This change is due to the fact that we are now analyzing the most recent stable release:

![image](https://user-images.githubusercontent.com/112710/101061894-ffd0bd00-355e-11eb-9e58-bf1903841104.png)